backend/src/models/question.js

```diff
@@ -26,6 +15,19 @@ class Question {
             .query('SELECT * FROM question WHERE id=?', [questionID])
     }
 
+    /**
+     * @param   {string} code        The code of the couse we're getting questions for.
+     * @param   {number} pageNumber  The pageNumber, defaults to 1?, if higher than max should just give max.
+     * @returns {object}
+     */
+    getQuestions(code, pageNumber) {
```

Comment:
Maybe a generic pagination interface could be used across models that query for multiple items from the database?

Reply:
I'm not really sure what you mean by this?

Reply:
I guess I meant having a function which structures paginated SQL queries for you... But now I realise it's probably out of scope of this issue; it should be done in a generic database layer if we end up adding that